docs/first-order/classify_waveforms.ipynb

```diff
@@ -0,0 +1,2327 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    electrodes = nwb.electrodes.to_dataframe()
</pre><p>this should be in a different cell. Don't you already cast it to a dataframe earlier?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>